docs-src/concepts/what-is-a-list-filter.md

```diff
@@ -31,19 +31,17 @@ A List Filter contains [Search Attribute](/concepts/what-is-a-search-attribute)
   - **BETWEEN ... AND**
   - **IN**
   - **ORDER BY**
-  - **LIKE**
-
-- Wildcards ('\*', '%', etc.) are supported for string-type List Filter Search Attributes.
-  Use wildcards with the **LIKE** operator to query possible values:
-
-  ```bash
-  // Create a "ProductId" custom Search Attribute of type String
-  tctl admin cluster add-search-attributes --name ProductId --type String
-  // Match values that start with "book"
-  ProductId LIKE "book%"
-  // Match values that contain "favorite"
-  ProductId LIKE "%favorite%"
-  ```
+
+  The last operator (**ORDER BY**) is supported only with Elasticsearch as the Visibility store.
```

Comment:
I would maybe instead of "Elasticsearch as Visibility store" put "Advanced Visibility" and link to docs page that describes advanced visibility, think there is a page on this right?

Reply:
@DJSanti - I like the sentence, but let's definitely link to `/clusters/how-to-integrate-elasticsearch-into-a-temporal-cluster`

@tsurdilo Can we assume then that `ORDER BY` is available with Cloud Visibility?